src/constructs/core/__data__/cfn.yaml

```diff
@@ -0,0 +1,10 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
`CfnInclude` will throw if it cannot find the YAML file to include in a CDK stack.

This file won't make it into the published module.